src/app/containers/InlineLink/index.jsx

```diff
@@ -32,13 +32,23 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
     );
   }
 
-  // else return a normal hyperlink
+  if (isExternal) {
+    return (
+      <InlineLink
+        href={locator}
+        aria-label={`${blocks[0].model.text}${externalLinkText}`}
+      >
+        <Blocks blocks={blocks} componentsToRender={componentsToRender} />
+        {isExternal ? (
+          <VisuallyHiddenText>{externalLinkText}</VisuallyHiddenText>
```

Comment:
Also, the VisuallyHiddenText component can be removed from here, since it is no longer required.